omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,59 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',
+    function ($scope,TagService,ngDialog) {
+
+       /**
+         * Method to load Tags
+         */
+        $scope.loadTags =function (personUuid) {
+            $scope.thisPersonUuid = personUuid;
```

Comment:
I'm a little confused, there is something that doesn't seem right with this personUuid, what's the point of copying personUuid into the scope if you are still passing it around when calling loadTags()? I would imagine that it's a scope variable and not just a variable local to the loadTags function that way you don't have to pass it around.